### PR TITLE
Feature/onboarding interest

### PR DESCRIPTION
**Pull request by shylajah15**
- Added UI screen for Onboarding Interest Page
- Added a success screen when user onboards
- Updated functionality for interest page based on acceptance criteria
- Made responsive for desktop versions(1024px)

### CODE REVIEW

File: src/components/Onboarding/Skills.vue [hunk 1]
```diff
@@ -68,11 +159,15 @@ export default {
     },
     roleSlug() {
       // TODO: Refactor the code
+      console.log(this.allAuth.profile.attributes.role_slugs);
```
Comment (gkmahendran09) [inline src/components/Onboarding/Skills.vue:162]: linting might fail here due to the console log, can you check and remove it,

**Review by gkmahendran09 — CHANGES_REQUESTED** [review]
Please check the console log and check for any linting errors

File: src/components/Onboarding/Skills.vue (hunk 1 above)
Comment (shylajah15) [reply src/components/Onboarding/Skills.vue]: Fixed this issue and checked the linting issues as well. Please review it again

**Review by gkmahendran09 — APPROVED** [review]
LGTM!

**Pull request merged**